javascripts/discourse/components/kanban/nav.gjs

```diff
@@ -0,0 +1,31 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import i18n from "discourse-common/helpers/i18n";
+import concatClass from "discourse/helpers/concat-class";
+
+export default class KanbanNav extends Component {
+  <template>
+    <li>
```

Comment:
The button should be rendered based on the `display_categories` setting.

This is a change forgotten from the old code which was using a connector:
```js
shouldRender(args) {
  return displayConnector(args.category?.get("slug"));
},
```